src/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,106 @@
+
+![Web tier scaling]
+
+This complements the vertical scaling provided by the core tier. As the core tier scales in size and power to accommodate a growing database and service usage, the web tier scales in size and power, and even instances, to accommodate growing process and traffic requirements.
+
+Consider an example that the web node instance type is _C5.2xlarge with 8 CPU and 16Gb RAM_. The number of requests to the site increased greatly. You can add an additional C5.2xlarge node to handle the increase in php-fpm processes or you can change each instance type to _C5.4xlarge with 16 CPU and 32Gb RAM_. Adding an additional node reduces the risk of insufficient surge capacity.
+
+## Project structure
+
+Minimally, Pro projects with the Scaled architecture have 6 nodes available.
+
+-  3 web nodes c5.2xlarge (8 CPU, 16 Gb RAM)
+-  3 core nodes m5.2xlarge (8 CPU, 32 Gb RAM)
+
+Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the New Relic Infrastructure service, which automatically connects with the application data and performance analytics to provide dynamic server monitoring. See [New Relic services][nri].
+
+### SSH access
+
+Certain files and logs, such as the `/app/<project-id>/var/log` directory, are not shared between nodes. Each node has a unique SSH access. You can not use the Magento Cloud CLI to log in to the core or web nodes, but you can find the node addresses in the [SSH Access list] in the Project Web UI.
+
+```bash
+ssh <node>.<project-ID>-<environment>-<user-ID>@ssh.<region>.magento.com
+```
+
+-  `node` 1 to 3—Addresses to access the core nodes
+-  `node` 4 to _n_—Addresses to access the web nodes
+
+Example response as you log in to a **core node** includes the _unified_ role:
+
+```terminal
+ __  __                   _          ___ _             _
+|  \/  |__ _ __ _ ___ _ _| |_ ___   / __| |___ _  _ __| |
+| |\/| / _` / _` / -_) ' \  _/ _ \ | (__| / _ \ || / _` |
+|_|  |_\__,_\__, \___|_||_\__\___/  \___|_\___/\_,_\__,_|
+            |___/
+
+ Welcome to Magento Cloud.
+
+ This is server unique-server-id, role project-id:unified.
+
+project-id@server-id:~$
+```
+{:.no-copy}
+
+Example response as you log in to a **web node** includes the _web_ role:
+
+```terminal
+ __  __                   _          ___ _             _
+|  \/  |__ _ __ _ ___ _ _| |_ ___   / __| |___ _  _ __| |
+| |\/| / _` / _` / -_) ' \  _/ _ \ | (__| / _ \ || / _` |
+|_|  |_\__,_\__, \___|_||_\__\___/  \___|_\___/\_,_\__,_|
+            |___/
+
+ Welcome to Magento Cloud.
+
+ This is server unique-server-id, role project-id:web.
+
+project-id@server-id:~$
+```
+{:.no-copy}
+
+### Log locations
+
+The log locations vary slightly depending on the node. For example, database logs are not written to web nodes.
```

Comment:
This is slightly confusing - I think you are saying that the available logs depend on the node type.  Will the logs have different paths depending on the node, or just some logs are available only on a specific node type? 

Guessing that this table will have more entries eventually.

Reply:
Yes. It should not be confusing. You won't see a database log on a node that doesn't contain a database.
In any case, this list may grow or it may be passed off to log locations topic. Not sure to the extent of this change yet. I created this section to accommodate a list similar to log locations, but haven't received much information.